compiler/one-cmds/tests/onecc_024.test

```diff
@@ -70,7 +70,7 @@ LUCI_LOG=5 onecc -C ${configfile} -OONECC_024 > ${filename}.log 2>&1
 
 clean_envir
 
-if grep -q "MakeBatchNormGammaPositivePass" "${filename}.log"; then
+if ! grep -q "MakeBatchNormGammaPositivePass" "${filename}.log"; then
```

Comment:
This change is necessary because now group options have lower priority than the options in user-given cfg file.

OONECC_024.cfg (group option)
```
make_batchnorm_gamma_positive=True
```

onecc_024.cfg (user option)
```
make_batchnorm_gamma_positive=False
```

The result is now False.